csrc/python_frontend/fusion_record.h

```diff
@@ -379,6 +380,78 @@ struct ReshapeOpRecord : RecordFunctor {
   std::vector<int64_t> new_shape_;
 };
 
+struct PadOpRecord : RecordFunctor {
+  PadOpRecord(
+      std::vector<State> _args,
+      std::vector<State> _outputs,
+      std::vector<State>& pad_widths)
```

Comment:
I'm questioning our choice of having `pad_widths` here as a separate thing, other than including it in `_args`. Is there any real benefits doing it this way?